openedx/core/djangoapps/api_admin/models.py

```diff
@@ -80,6 +80,29 @@ def api_access_status(cls, user):
         except cls.DoesNotExist:
             return None
 
+    @classmethod
+    def retire_user(cls, user):
+        """
+        Retires the user's API acccess request table for GDPR
+        
+        Arguments:
+            user (User): The user linked to the data to retire in the model.
+
+        Returns:
+            True: If the user has a linked data in the model and retirement is successful
+            False: user has no linked data in the model.
+        """
+        try:
```

Comment:
I'd write this as

```
try:
    retire_target = cls.objects.get(user=user)
except cls.DoesNotExist:
    return False
else:
    retire_target.website = ''
    retire_target.company_address = ''
    retire_target.company_name = ''
    retire_target.reason = ''
    retire_target.save()
```

Reply:
...because "Effective Python" told me to. Consider it a nit.

Reply:
Your suggestion have syntax error. You cannot have `else` without an `if`.
The reason we are doing this way is because `user` have an one on one relationship with this table. `filter()` call won't be necessary.

Reply:
It shouldn't be, I think:
<img width="483" alt="screen shot 2018-05-11 at 11 32 59" src="https://user-images.githubusercontent.com/4751297/39932799-237df9bc-550f-11e8-9daf-ef12c7d2c480.png">

But, again, the suggestion shouldn't block you. I was curious to hear your impression of the code style.

Reply:
huh, i didn't know. I can explore a bit this style.